blackvuesync.py

```diff
@@ -588,6 +596,12 @@ def parse_args():
                                  "from oldest to newest; ""rdate"": downloads in chronological order "
                                  "from newest to oldest; ""type"": prioritizes manual, event, normal and then parking"
                                  "recordings; defaults to ""date""")
+
+    arg_parser.add_argument('-f', "--filter", default=None,
+                            help="specify which event you want to download"
+                                 " ie: -f PF PR returns only Parking Front and Parking Rear events",
```

Comment:
we need single quotes here (and other small stuff if you don't mind): `e.g.: --filter 'PF PR' downloads only Parking Front and Parking rear even recordings`

Reply:
if you use `--filter 'PF PR'`  `args.filter` will look like a string `'PF PR'`  `nargs='*'`parses it correctly on its own